docs/advanced_examples/DecisionTreeClassifier.ipynb

```diff
@@ -300,8 +312,8 @@
      "output_type": "stream",
      "text": [
       "Ground truth:       [0 0 0 1 0 1 0 0 0 0]\n",
-      "Prediction sklearn: [0 0 0 1 0 1 0 0 0 0]\n",
-      "Prediction FHE:     [0 0 0 1 0 1 0 0 0 0]\n"
+      "Prediction sklearn: [0 0 0 0 0 0 0 1 0 0]\n",
+      "Prediction FHE:     [0 0 0 0 0 0 0 1 0 0]\n"
```

Comment:
Hmm that's bad.

Reply:
We should probably seed notebooks?

Reply:
I am pretty sure this is seeded. At least, these results never changed since Jan 2022 so something changed here.

Reply:
I reverted the notebook